cmd/ktranslate/main.go

```diff
@@ -56,6 +56,8 @@ func main() {
 		teeLog         = flag.Bool("tee_logs", false, "Tee log messages to sink")
 		appMap         = flag.String("application_map", "", "File containing custom application mappings")
 		syslog         = flag.String("syslog.source", "", "Run Syslog Server at this IP:Port or unix socket.")
+		httpInput      = flag.String("http.source", "", "Listen for content sent via http. telegraph")
```

Comment:
Is this only telegraph?

Reply:
I guess there's 2 parts to this. 1) is listen on http for data posted in. 2) is deserialize into a known struct. Right now http_input only knows how to listen for telegraph inputs, but it would be easy enough to add more logic to handle other data formats as needed.